### PR TITLE
Feature/create product order dish entities

### PR DESCRIPTION
**Pull request by gmzjohn**
Create entities for product, dish, and order

### CODE REVIEW

**Comment by gmzjohn** [comment]
So since all properties are private I think we can implement the constructor to set the properties there and also implement a getter for each property, example:

image

**Comment by reynoldmorel** [comment]
> So since all properties are private I think we can implement the constructor to set the properties there and also implement a getter for each property, example:
> 
> image

Why not just using `public` variables?

**Comment by reynoldmorel** [comment]
> etters needed that way? Or we are going to leave it for readability?

you don't need getters that way

**Comment by gmzjohn** [comment]
> > etters needed that way? Or we are going to leave it for readability?
> 
> 
> 
> you don't need getters that way

How about the constructors? I believe they are needed right?

**Comment by reynoldmorel** [comment]
> > > etters needed that way? Or we are going to leave it for readability?
> > 
> > 
> > you don't need getters that way
> 
> How about the constructors? I believe they are needed right?

yes they are

File: entities/Dish.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default class Dish {
+    id; // int
```
Comment (reynoldmorel) [inline entities/Dish.js:2]: I think all these properties here are not necessary just leave the constructor definition

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**